llvm/lib/Target/AIE/AIE2InstrInfo.td

```diff
@@ -557,7 +557,7 @@ def LoopJNZ :
     Pseudo<(outs), (ins eR:$tc, eP:$target), "loop_jnz", "${tc}, ${target}">;
 
 // Zero overhead loops
-let hasSideEffects = 1, mayLoad = 0, mayStore = 0 in
+let hasSideEffects = true, mayLoad = 0, mayStore = 0, isNotDuplicable = true in
```

Comment:
Super nit: While you're at it, also change the `0`s to `false`?

Reply:
Yes!